wagtail_newsletter/models.py

```diff
@@ -1,7 +1,10 @@
-from typing import Optional
+from typing import Any, Optional
 
 from django.core.exceptions import ValidationError
 from django.db import models
+from django.http import HttpResponse
+from django.template.loader import render_to_string
+from django.utils.safestring import SafeString
```

Comment:
I think that syntax is meant to avoid importing heavy modules unnecessarily. TBH I avoid it, because it adds noise to the start of the module, for (I think) little practical benefit.